bayesrules/environment.yml

```diff
@@ -0,0 +1,14 @@
+name: bayesrules
```

Comment:
Added a much stricter environment though I see I need to add pyreadr and watermark to it. I'll make the update

Reply:
I also need wget or request to download the R data files. Could you include one of them ?

Reply:
Also I can't create this environment:
(base) lode@datarootss-MacBook-Pro-2 resources % cd Bayes_Rules/  
(base) lode@datarootss-MacBook-Pro-2 Bayes_Rules % conda env create -f environment.yml 
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - libzip==1.5.1=h8d318fa_1003
  - hdf4==4.2.13=h3ca952b_2
  - sniffio==1.2.0=py39h06a4308_1
  - matplotlib==3.5.1=py39h06a4308_1
  - jedi==0.18.1=py39h06a4308_1
  - numexpr==2.8.1=py39h6abb31d_0

Reply:
Apologies on that, ill get this fixed in next 24 hours

Reply:
When you get a chance can you try again? https://github.com/pymc-devs/resources/pull/181/files

Reply:
I tried but again i get ResolvePackageNotFound....

Reply:
oh man, im sorry, environment setup is the worst. I hate to hold you up. Lets do this we can merge as is and then I'll figure out the environment outside of this PR. Thanks for being with me. I can get back to a review on this in the next day

Reply:
No problem. Agree to proceed as you propose. I'm sure to resolve the environment issues. I use environment_simple.yml for the moment, so no holding up.